CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for regular expressions to split pre-tokenizer. See
   `Tokenizers.PreTokenizer.split_regex/3`.
 
+### Removed
+
+- **(Breaking)** `:add_prefix_space` option in favour of `:prepend_scheme` for metaspace
+  decoder and pre-tokenizer
```

Comment:
Technically it's a small change, but I think it's a good idea to do 0.5.0 anyway, because we bumped hf/tokenizers by a couple versions since the last release.